ports/espressif/boards/waveshare_esp32s2_pico/board.h

```diff
@@ -0,0 +1,56 @@
+/* 
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 Ha Thach (tinyusb.org) for Adafruit Industries
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#ifndef WAVESHARE_ESP32_S2_PICO_
+#define WAVESHARE_ESP32_S2_PICO_
+
+//--------------------------------------------------------------------+
+// Button
+//--------------------------------------------------------------------+
+
+#define PIN_BUTTON_UF2        0
+
+//--------------------------------------------------------------------+
+// LED
+//--------------------------------------------------------------------+
+
+
+#define LED_PIN               9
+#define LED_STATE_ON          0
+
+//--------------------------------------------------------------------+
+// USB UF2
+//--------------------------------------------------------------------+
+
+#define USB_VID           0x2E8A
+#define USB_PID           0x101F
```

Comment:
is this vid/pid allocated for this board to use with tinyuf2. If yes, would you mind give a reference link to vid/pid

Reply:
I used the vid and pid from:
https://github.com/adafruit/circuitpython/blob/main/ports/raspberrypi/boards/waveshare_rp2040_zero/mpconfigboard.mk

As I could not find "Waveshare Electronics" here:
https://www.usb.org/sites/default/files/vendor_ids063022_0.pdf
Or in the manufacturer's wiki:
https://www.waveshare.com/wiki/ESP32-S2-Pico

The demo codes they [provide](https://www.waveshare.com/w/upload/6/69/ESP32-S2-Pico_Code.zip) use other board's firmwares and uf2's, so I can't use those as a ref...

I do not have any experience with this and I do not know what to do other than to provide waveshare_RP2040_zero's id's.

Reply:
I have created a support ticket to waveshare.

Reply:
thanks, let wait for the confirmation, in general, each product should have its own vid/pid.

Reply:
Today, I recieved the reply.
USB_VID: 239a
USB_PID: 00a5
I have commited them to the pr.